src/containers/Admin/LogoTab/Description/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect, FormEvent } from 'react'
+
+import { Label, Heading3, Paragraph } from '@entur/typography'
+import { TextField } from '@entur/form'
+
+import { useDebounce } from '../../../../utils'
+import { useSettingsContext } from '../../../../settings'
+
+const Description = (): JSX.Element => {
+    const [{ description, logoSize }, { setDescription }] = useSettingsContext()
+
+    const [value, setValue] = useState(description)
+
+    const debouncedValue = useDebounce(value, 1000)
+
+    useEffect(() => {
+        if (description !== debouncedValue) {
+            setDescription(debouncedValue)
+        }
+    }, [description, debouncedValue, setDescription])
+
+    return (
+        <>
+            <Heading3>Legg til beskrivelse</Heading3>
+            <Paragraph>
+                Her kan du skrive inn en beskrivelse av tavla. Beskrivelsen er
+                plassert under logoen oppe i venstre hjørne. Dette feltet kan
+                brukes til å beskrive avgangene man ser, hva stoppestedene
+                ligger i nærheten av eller henvise til andre kanaler. Maks 80
+                tegn.
+            </Paragraph>
+            <Label>Beskrivelse av avgangstavla</Label>
+            <TextField
+                value={value}
+                onChange={(event: FormEvent<HTMLInputElement>): void =>
+                    setValue(event.currentTarget.value)
+                }
+                maxLength={80}
+                disabled={logoSize === '56px'}
```

Comment:
Det er litt for lite synlig at denne er disabled, man kan fort tro at man skal kunne skrive inn noe og at noe er galt.

Reply:
Jeg er enig, men vet ikke hva vi burde gjøre med det. Fikk bestilt noen endringer i designsystemet, så fargen skal i hvert fall bli bedre. Trenger vi legge til noen form for melding?

Reply:
Tenker egentlig det, for det kan være vanskelig å vite _hvorfor_ noe er disabled til å begynne med. Egentlig synes jeg at man skal kunne endre description uansett hvilket valg man har gjort på størrelse, og så heller gjøre det klart at beskrivelsen blir midlertidig borte når man har stor logo.